src/tcpedit/tcpedit_opts.def

```diff
@@ -58,6 +58,49 @@ you can reuse the same seed value to recreate the traffic.
 EOText;
 };
 
+flag = {
+    name        = range_count_src;
+    descrip     = "Set the count of source ip address range";
+    arg-type    = number;
+    arg-range   = "1->1048575";
+    arg-default = 1;
+    max         = 1;
+    doc         = <<- EOText
+Increament source ip address (ipv4) within specified range.
+EOText;
+};
+
+flag = {
+    name        = range_count_dst;
+    descrip     = "Set the count of destination ip address range";
+    arg-type    = number;
+    arg-range   = "1->1048575";
+    arg-default = 1;
+    max         = 1;
+    doc         = <<- EOText
+Increament destination ip address (ipv4) within specified range.
+EOText;
+};
+
+
+flag = {
+    name        = range_address;
+    arg-type    = string;
+    max         = 1;
+    descrip     = "Rewrite IP addresses to be src/dst";
+    doc         = <<- EOText
+Takes a pair of colon delimited IPv4 addresses which will be used to rewrite
+all traffic to appear to be based on the src/dst IP addresses.
+
+IPv4 Example:
+@example
+--endpoints=198.18.50.2:198.19.0.2
```

Comment:
@dsugisawa-mixi  Is this a typo? Please correct and create PR that merges to this branch.

Also, the help has to explain what this set of parameters does. We need more than just how to use the functions. Without looking at the code, I cannot understand what they do, and which functions rely on each other.

Maybe have a look at `--pnat` as an example. Also, have a look at it to see if there is overlap.